client/components/scrollable-horizontal-navigation/index.tsx

```diff
@@ -0,0 +1,129 @@
+import { Button, Gridicon, SegmentedControl } from '@automattic/components';
```

Comment:
All the code has been extracted from `client/reader/discover/discover-navigation.js`.